### PR TITLE
testng-eclipse#124: update doc for eclipse plugin

### PR DESCRIPTION
**Pull request by missedone**
update docs for https://github.com/cbeust/testng-eclipse/issues/124
1) launch configuration options
2) workbench preferences
3) project properties

### CODE REVIEW

File: doc/eclipse.html [hunk 1]
```diff
@@ -44,6 +44,12 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 <br>
 &nbsp;&nbsp;<a class="summary" href="#eclipse-quickfixes">8 - Quick fixes</a>
 <br>
+&nbsp;&nbsp;<a class="summary" href="#eclipse-preferences">9 - Preferences and Properties</a>
+<br>
+&nbsp;&nbsp;<a class="summary" href="#eclipse-workbench-preferences">9.1 - Workbean Preferences</a>
```
Comment (cbeust) [inline doc/eclipse.html]: Did you mean "Workbench"?

File: doc/eclipse.html [hunk 2]
```diff
@@ -87,12 +93,26 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 <img border="0" src="http://testng.org/pictures/launch.png" />
 </p>
 <p>You should change the name of this configuration and pick a project, which 
-can be selected by clicking on the <tt>Browse...</tt> button at the top of the window.</p><p>Then you choose to launch your TestNG tests in the following ways:</p><h4><a name="eclipse-launch-class">4.2.1 - From a class file</a></h4><p>Make sure the box near <tt>Class</tt> is checked and then pick a class from your 
+can be selected by clicking on the <tt>Browse...</tt> button at the top of the window.</p>
+<p>
+<ul>
+Runtime options:
```
Comment (cbeust) [inline doc/eclipse.html]: Above the ul I think?

File: doc/eclipse.html [hunk 3]
```diff
@@ -87,12 +93,26 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
...
+<li><b>Log Level</b>: specify the value (0-10) for different verbose log level</li>
```
Comment (cbeust) [inline doc/eclipse.html]: levels

File: doc/eclipse.html [hunk 4]
```diff
@@ -87,12 +93,26 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
...
+<li><b>Verbose</b>: check it to enable the runtime TestNG verbose log</li>
```
Comment (cbeust) [inline doc/eclipse.html]: You can remove "check it to"

File: doc/eclipse.html [hunk 5]
```diff
@@ -87,12 +93,26 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
...
+<li><b>Debug</b>: check it to enable more runtime TestNG debug info</li>
```
Comment (cbeust) [inline doc/eclipse.html]: Remove "check it ti"

File: doc/eclipse.html [hunk 6]
```diff
@@ -87,12 +93,26 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
...
+<li><b>Serialization Protocol</b>: the serialization protocol used for commuincating between TestNG Eclipse Plugin and TestNG runtime.
```
Comment (cbeust) [inline doc/eclipse.html]: communicating

File: doc/eclipse.html [hunk 7]
```diff
@@ -87,12 +93,26 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
...
+By default it's "Object Serialization", the other option is "String Serializatio" but is deprecated.</li>
```
Comment (cbeust) [inline doc/eclipse.html]: The default is "Object serialization" (the other option, "String serialization" is deprecated).

File: doc/eclipse.html [hunk 8]
```diff
@@ -87,12 +93,26 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
...
+<li><b>Add the POM's JVM Arguments when launch</b>: for maven based project, we normally define maven-surefire-plugin for test phase and maven-failsafe-plugin for integration test phase, there might be having the testing runtime JVM arguments defined in the "argLine" element of the maven plugin configuration. If you check this option, TestNG Eclipse Plugin will try to recognize the JVM arguments in "argLine", add them to TestNG runtime JVM arguments when lauch.<br/>
```
Comment (cbeust) [inline doc/eclipse.html]: when launched

File: doc/eclipse.html [hunk 9]
```diff
@@ -87,12 +93,26 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
...
+Be note that, if there's multiple "argLine" element in the pom.xml, the first presence will be used. If it's not the desired behaviour, you can simply uncheck this option, and type your JVM arguments on the "Arguments" tab next to this "Test" tab.</li>
```
Comment (cbeust) [inline doc/eclipse.html]: I'll resume reviewing this PR from here later.

**Comment by missedone** [comment]
updated, thanks

File: doc/eclipse.html [hunk 10]
```diff
@@ -54,11 +60,14 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 / Other...</tt>&nbsp; and you should see the TestNG view listed in the Java category.</p><p align="center">
 <img border="0" src="http://testng.org/pictures/view.png" />
 </p>
+<p>
+NOTE: since TestNG Eclipse Plugin 6.9.8, the minimum required TestNG version is <b>6.5.1</b>
```
Comment (missedone) [inline doc/eclipse.html]: state the minimum required TestNG version supported by plugin

File: doc/eclipse.html [hunk 11]
```diff
@@ -87,12 +96,27 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
...
-can be selected by clicking on the <tt>Browse...</tt> button at the top of the window.</p><p>Then you choose to launch your TestNG tests in the following ways:</p><h4><a name="eclipse-launch-class">4.2.1 - From a class file</a></h4><p>Make sure the box near <tt>Class</tt> is checked and then pick a class from your 
+can be selected by clicking on the <tt>Browse...</tt> button at the top of the window.</p>
+<p>
+Runtime options:
+<ul>
+<li><b>Log Level</b>: specify the value (0-10) for different verbose log levels</li>
+<li><b>Verbose</b>: enable the runtime TestNG verbose log</li>
+<li><b>Debug</b>: enable more runtime TestNG debug info</li>
+<li><b>Serialization Protocol</b>: the serialization protocol used for communicating between TestNG Eclipse Plugin and TestNG runtime.
+By default it's "Object Serialization", the other option is "String Serializatio" but is deprecated.</li>
```
Comment (cbeust) [inline doc/eclipse.html]: "The default is "Object Serialization", the other option ("String Serialization") is deprecated.

**Comment by missedone** [comment]
updated

**Pull request merged**